services/contentapi/main.py

```diff
@@ -0,0 +1,18 @@
+from fastapi import FastAPI
+from pydantic import BaseModel
+
+app = FastAPI(
+    title="RAISE Content API"
+)
+
+
+class ContentData(BaseModel):
+    id: str
+    content: str
+
+
+@app.get("/contents/{content_id}", response_model=ContentData)
+async def create_event(content_id):
+    content = f"<div><p>This is content for ID {content_id}</p></div>"
+    data = ContentData(id=content_id, content=content)
+    return data
```

Comment:
I'm not entirely sure why these files are being included. I created this branch off the previous branch I was working on for this issue, so I could see how they could be marked as new to main... But before I pushed I merged the updated main into this branch. Any ideas? @rnathuji

Reply:
From a brief glance, it looks like the corresponding commit sha on this branch is different than the one on `main`. Perhaps you force pushed a change before merging :man_shrugging: ?

In any case, you can probably `rebase` against main and then force push the updated branch up.